cli/test/e2e/package_repo_release_test.go

```diff
@@ -8,52 +8,68 @@ import (
 	"strings"
 	"testing"
 
+	uitest "github.com/cppforlife/go-cli-ui/ui/test"
 	"github.com/stretchr/testify/require"
 )
 
 const (
-	packagesDir = "packages"
+	packagesDir       = "packages"
+	pkgRepoOutputFile = "package-repository.yml"
+	pkgrName          = "testpackagerepo.corp.dev"
 )
 
 func TestPackageRepositoryReleaseInteractively(t *testing.T) {
-	env := BuildEnv(t)
-	logger := Logger{}
-	kappCtrl := Kctrl{t, env.Namespace, env.KctrlBinaryPath, logger}
-
 	cleanUp := func() {
 		os.RemoveAll(workingDir)
 	}
 	cleanUp()
 	defer cleanUp()
 
-	err := os.Mkdir(workingDir, os.ModePerm)
-	if err != nil {
-		t.Fatalf("Unable to create working directory: %s", err.Error())
-	}
+	env := BuildEnv(t)
+	logger := Logger{}
+	kappCtrl := Kctrl{t, env.Namespace, env.KctrlBinaryPath, logger}
+	kappCli := Kapp{t, env.Namespace, env.KappBinaryPath, logger}
```

Comment:
kappCtrl -> kctrl
kappCli -> kapp

Also, any specific reason to move these down?

Reply:
No, just moved cleanup up.